scripts/cli.rb

```diff
@@ -1,5 +1,6 @@
 require 'rubygems'
 require 'trollop'
+require_relative 'target'
 
 module Build
   class Cli
```

Comment:
Thanks for adding the `Cli` class and parse. I've been trying to follow this trend with `ApplianceConsole::Cli` and `FixAuth::Cli`